### ISSUE TITLE
RCS1163 False Positive

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#251]
This code throws a false positive RCS1163.

```csharp
public static void Method(DataGridView datagridview)
{
    var columns = from c in datagridview.Columns.Cast<DataGridViewColumn>() select c;
}
```
